action.yaml

```
@@ -1,5 +1,5 @@
-name: mpyl
-description: Action to run mpyl from a docker container
+name: MPyL
```

```suggestion
name: Run MPyL
```

I'd be a little more descriptive on the name